### PR TITLE
Add a Static Pages controller

### PR DESCRIPTION
**Pull request by hoanganhh**
Finish static pages

### CODE REVIEW

**Comment by hoanganhh** [comment]
ready

**Comment by hoanganhh** [comment]
ready

File: app/views/static_pages/help.html.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<h1>Help</h1>
```
Comment (thangtx88) [inline app/views/static_pages/help.html.erb]: I18n

File: app/views/static_pages/home.html.erb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<h1>Sample App</h1>
```
Comment (thangtx88) [inline app/views/static_pages/home.html.erb]: I18n

**Comment by thangtx88** [comment]
Commented

**Comment by hoanganhh** [comment]
ready

**Pull request merged**